etcdserver/api/v3rpc/header.go

```diff
@@ -38,7 +38,7 @@ func newHeader(s *etcdserver.EtcdServer) header {
 // fill populates pb.ResponseHeader using etcdserver information
 func (h *header) fill(rh *pb.ResponseHeader) {
 	if rh == nil {
-		plog.Panic("unexpected nil resp.Header")
+		panic("unexpected nil resp.Header")
```

Comment:
Note: same here, not sure how to pass in zap Logger.

Reply:
Think just `panic` is fine.